docs/modules/plugins/partials/proxy-steps.adoc

```diff
@@ -0,0 +1,283 @@
+----
+
+==== Capture HTTP message
+
+Save the HTTP message part from the HTTP request with given URL-pattern into the variable with specified name and the scopes.
+
+{proxy}
+
+The actions preformed by the step:
+
+* extract HTTP messages from the recorded proxy archive
+* filter out the HTTP messages with the response status code `302 Moved Temporarily`
+* find HTTP requests matching the provided HTTP methods and the URL regular expression
+* check that total number of the found HTTP messages is equal to `1`
+* save the HTTP message part to the specified variable
+
+{har-attached}
+
+[source,gherkin]
+----
+When I capture HTTP $httpMethods request with URL pattern `$urlPattern` and save $httpMessagePart to $scopes variable `$variableName`
+----
+* `$httpMethods` - The "or"-separated set of HTTP methods to filter by, e.g. `GET or POST or PUT`.
+* `$urlPattern` - The regular expression to match HTTP request URL.
+* `$httpMessagePart` - The HTTP message part to save. One of:
+** `URL` - The request URL.
+** `URL query` - The request URL query parameters.
+** `request data` - The request data includes the following keys:
+*** `query` - The URL query is stored as a collection of key and value pairs, where key is the name of the query parameter and value is the list of query parameter values. The query parameter values are accessible via zero-based index.
+*** `requestBody.mimeType` - The MIME type of posted data, the variable will not be saved if MIME type is not present.
+*** `requestBody.text` - The posted data as plain text, the variable will not be saved if the request body is not present.
+*** `requestBodyParameters` - The form data parameters are stored as a collection of key and value pairs, where key is the name of the form parameter and value is the list of form parameter values. The form parameter values are accessible via zero-based index.
+*** `responseStatus` - The response status, the variable will not be saved if the response is not present.
+** `response data` - The response data includes the following keys:
+*** `responseBody.mimeType` - The MIME type of response data, the variable will not be saved if MIME type is not present.
+*** `responseBody.text` - The response data as plain text, the variable will not be saved if the response body is not present.
+* `$scopes` - xref:commons:variables.adoc#_scopes[The comma-separated set of the variables scopes].
+* `$variableName` - The variable name to save the HTTP message part.
+
+.Validate the URL of the matching HTTP request
+[source,gherkin]
+----
+Given I am on a page with the URL 'https://www.google.com/search?q=vividus'
+When I capture HTTP GET or POST request with URL pattern `.*/search.*=vividus` and save URL to scenario variable `URL`
+Then `${URL}` is equal to `https://www.google.com/search?q=vividus`
+----
+
+.Validate the URL query of the matching HTTP request
+[source,gherkin]
+----
+Given I am on a page with the URL 'https://www.google.com/search?q=vividus'
```

Comment:
these examples are not applicable for mobile apps

Reply:
{givent-step}